### ISSUE TITLE
inconsistency between the documentation and implementation of use_ndim

### ISSUE THREAD

**Issue by aras-y** [wannesm/dtaidistance#197]
For multivariate time series (`use_ndim=True`), there is an inconsistency between the documentation and the implementation. 

The [documentation](https://dtaidistance.readthedocs.io/en/latest/modules/dtw.html?highlight=warping_paths#dtaidistance.dtw.warping_paths) and [the docstring](https://github.com/wannesm/dtaidistance/blob/0d1b41782aaabc4c05a1722dd8379f35939a6d55/dtaidistance/dtw.py#L417) says Euclidean distance but the implementation (both in the pip (2.3.10) and Github versions) uses squared Euclidean distance by default (as it should be, for consistency with the squared difference in the univariate case). 

So, I suggest the documentation to be updated to the squared Euclidean distance.

**Comment by kingwe-stack** [comment]
That's right,I also found that.I suggest to add Euclidean distance calculation function .

**Comment by wannesm** [comment]
Fixed by @aras-y in 4b23b99da4400025e52804bf683a3146a721b1d3 .

Euclidean distance is supported using the `inner_dist` argument.
